### PR TITLE
feat(route): add zyw今日热榜

### LINKED ISSUE #12525: 请帮我制作：热榜RSS
### Category

New media

### Website URL

https://hot.zyw.asia/

### Website description

这是一个实时刷新的各大网站热榜聚合站点

### What content should be included?

应该包含站点内聚合的各大网站热榜信息

### Additional description

_No response_

### This is not a duplicated rss request

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) and [pull requests](https://github.com/DIYgod/RSSHub/pulls) to ensure this rss proposal has not already been requested

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved Issue

Close #12525

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/zyw/hot
/zyw/hot/哔哩哔哩
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/zyw/hot.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+                for (const d of detailResponse.data.data) {
+                    items.push({
+                        link: d.url,
+                        title: d.title,
+                        description: d.desc,
+                    });
+                }
+
+                return;
```
Comment (TonyRL) [inline lib/v2/zyw/hot.js:55]: Since there's no return value, nothing will be stored in cache.
docs: https://docs.rsshub.app/joinus/use-cache.html

Comment (nczitzk) [reply lib/v2/zyw/hot.js:55]: Thanks for the review. I will remove `cache.tryGet` 🤝

File: lib/v2/zyw/radar.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'zyw.asia': {
+        _name: 'zyw',
+        hot: [
+            {
+                title: '今日热榜',
+                docs: 'https://docs.rsshub.app/new-media.html#zyw-jin-ri-re-bang',
+                source: ['/#/list', '/'],
```
Comment (TonyRL) [inline lib/v2/zyw/radar.js:8]: https://github.com/DIYgod/RSSHub/actions/runs/5005846277/jobs/8970349178?pr=12527#step:6:19

Comment (nczitzk) [reply lib/v2/zyw/radar.js:8]: 👌

File: lib/v2/zyw/radar.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
```
Comment (TonyRL) [inline lib/v2/zyw/radar.js:8]:
```suggestion
                source: ['/'],
```
https://github.com/DIYgod/RSSHub/actions/runs/5015935242/jobs/8992161258?pr=12527#step:6:16

**Pull request merged**